src/content/posts/2024/07/16/empower-teams-to-build-better-apps-faster.mdx

```diff
@@ -0,0 +1,75 @@
+stretching Tuist's original focus around Xcode's complexities.
+This made organizations and developer confusing about what Tuist is and what it does.
+Therefore, we decided to revisit our mission and vision and share it with all of you.
+
+## From idea to top-notch apps
+
+Apps usually start with an idea or a business need. Ideas can fade away if they're not validated quickly, and potentially great apps may never see the light of day. Unfortunately, the Apple platform has grown in complexity over the years, creating challenges and friction that developers and organizations need to navigate. Luckily, the community has built excellent open-source tools to assist developers with various challenges they face. However, developers still have to do the job of gluing these tools together into workflows that are a joy to use, sometimes in programming languages they are not familiar with, and maintain them indefinitely.
+
+We believe there must be a better way: an ["omakase"](https://en.wikipedia.org/wiki/Omakase) for app developers. We envision an integrated experience with sensible defaults and the right extensibility points. This experience would hold your hand from the moment you start your project until you ship it to your users, helping you scale your development and your team. It would be an experience that can be easily ported from one project to another, from one team to another, and from one organization to another. We are building that experience with Tuist.
+
+**We are shifting our focus to empower teams to build better apps faster.**
+
+## Meeting developers where they are
+
+Our strategy has always been to meet developers where they are by integrating with the official tools and APIs provided by Apple and equipping them with superpowers. A good testament to this work is our caching feature, which leverages Xcode primitives to achieve this. We will continue this approach, minimizing abstractions as much as possible, aligning with Apple's direction, and embracing the languages and tools that developers are familiar with and love, such as [Swift](https://www.swift.org/).
+
+Moreover, we are taking the approach of meeting developers where they are even further. We believe that ignoring the fact that developers spend significant time on platforms like [Slack](https://slack.com), [GitHub](https://github.com), and [GitLab](https://gitlab.com) is a mistake. Developers often spend a substantial portion of their time there collaborating with each other and with other teams and roles. It is crucial that their workflows integrate seamlessly with these platforms so that anyone in the organization can access the information they need to make informed decisions without having to install developer-specific toolchains. Therefore, we are embracing the web platform and URLs as units of collaboration. Developers and organizations that want to extend Tuist's capabilities further will be able to do so by plugging their projects into a server, and voila.
```

Comment:
```suggestion
Moreover, we are taking the approach of meeting developers where they are even further. We believe that ignoring the fact that developers spend significant time on platforms like [Slack](https://slack.com), [GitHub](https://github.com), and [GitLab](https://gitlab.com) is a mistake. Developers often spend a substantial portion of their time there collaborating with each other and with other teams and roles. It is crucial that their workflows integrate seamlessly with these platforms, so that anyone in the organization can access the information they need to make informed decisions without having to install developer-specific toolchains. Therefore, we are embracing the web platform and URLs as units of collaboration. Developers and organizations that want to extend Tuist's capabilities further will be able to do so by plugging their projects into a server, and voila.
```